.travis.yml

```diff
@@ -1,29 +1,16 @@
-language: python
-
-addons:
-  postgresql: "9.5"
-
-python:
-
-  - "2.7"
-
-env:
-
-- DJANGO_VERSION=1.9.7
+language: ruby
```

Comment:
Just for my understanding, why `ruby` here?

Reply:
It's the default.